api/v1beta1/channel_webhook.go

```diff
@@ -208,3 +215,23 @@ func validatePeersOwnership(ctx context.Context, c client.Client, ownerOrgs []st
 	}
 	return nil
 }
+
+func validateChannelID(ctx context.Context, c client.Client, network, channelID string) error {
+	selector := labels.NewSelector()
+	req, _ := labels.NewRequirement(CHANNEL_NETWORK_LABEL, selection.In, []string{network})
+	selector = selector.Add(*req)
+	listOption := client.ListOptions{
+		LabelSelector: selector,
+	}
+	channelList := &ChannelList{}
+	if err := c.List(ctx, channelList, &listOption); err != nil {
+		return err
+	}
+	for _, ch := range channelList.Items {
+		klog.V(5).Infof("by selector %s, get ch %s\n", selector.String(), ch.GetName())
```

Comment:
At present, the V series functions do not seem to be effective. Should we temporarily remove the V function?